doc/source/visualizing/geographic_projections_and_transforms.rst

```diff
@@ -0,0 +1,145 @@
+.. _geographic_projections_and_transforms:
+
+Geographic Projections and Transforms
+=====================================
+
+Geographic data that is on a sphere can be visualized by projecting that data
+onto a representation of that sphere flattened into 2d space. There exist a
+number of projection types, which can be found in the `the cartopy
+documentation <https://scitools.org.uk/cartopy/docs/latest/crs/projections.html>`_.
+With support from `cartopy <https://scitools.org.uk/cartopy/docs/latest/>`_, 
+``yt`` now supports these projection 
+types for geographically loaded data.
+Underlying data is assumed to have a transform of `PlateCarree
+<https://scitools.org.uk/cartopy/docs/latest/crs/projections.html#platecarree>`_,
+which is data on a flattened, rectangular, latitude/longitude grid. The data
```

Comment:
Maybe just a parenthetical note that typically this is the format that data arrives in.

Reply:
Addressed in b5c2d96e8016efa4d251755874cbe1e82a26f702